exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,8 @@
 // Start by creating a variable `message`
-
+let name = 'Sharm';
+let nameLength = name.length;
+const message = 'My name is ' + `${name}` + ' and my name is ' + `${nameLength}` + ' characters long'
```

Comment:
Unfortunately not the correct implementation of string literal syntax here, try playing around with having more than one variable in a string and you might see the usefulness of them...

Let me know if you're still unsure